app/src/main/java/com/realityexpander/tasky/agenda_feature/util/AgendaUtil.kt

```diff
@@ -0,0 +1,11 @@
+package com.realityexpander.tasky.agenda_feature.util
+
+import com.realityexpander.tasky.core.util.UuidStr
+
+typealias EventId = UuidStr
+typealias ReminderId = UuidStr
+typealias TaskId = UuidStr
+typealias AttendeeId = UuidStr
+typealias PhotoId = UuidStr
+
+typealias TimeZoneStr = String  // ex: "America/Los_Angeles"
```

Comment:
Cool, I like this to make the type of ID even clearer

Reply:
Thank you!